src/Appwrite/Platform/Workers/Deletes.php

```diff
@@ -565,10 +572,17 @@ private function deleteProject(Database $dbForConsole, callable $getProjectDB, D
         ], $dbForConsole);
 
         // Delete metadata table
-        if (!\in_array($dsn->getHost(), $sharedTables)) {
+        if ($projectTables) {
             $dbForProject->deleteCollection(Database::METADATA);
-        } else {
+        } elseif ($sharedTablesV1) {
             $this->deleteByGroup(Database::METADATA, [], $dbForProject);
+        } elseif ($sharedTablesV2) {
+            $queries = \array_map(
+                fn ($id) => Query::notEqual('$id', $id),
```

Comment:
shouldn't we do this by Internal ID?

Reply:
These are all the collections from `collections.php`, we can't get their internal IDs without fetching by ID first so it seems redundant